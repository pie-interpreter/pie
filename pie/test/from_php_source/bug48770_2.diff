001+ string(27) "B::func2: This should work!"
002+ string(27) "B::func3: This should work!"
003+ 
004+ Warning: call_user_func_array() expects parameter 1 to be a valid callback, cannot access private method B::func22() in /home/sery0ga/work/php-src/Zend/tests/bug48770_2.php on line 22
001- %unicode|string%(27) "A::func2: This should work!"
002- %unicode|string%(27) "A::func3: This should work!"
004- Warning: call_user_func_array() expects parameter 1 to be a valid callback, cannot access private method A::func22() in %s on line %d
006+ Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'B' does not have a method 'inexistent' in /home/sery0ga/work/php-src/Zend/tests/bug48770_2.php on line 23
005- 
006- Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'A' does not have a method 'inexistent' in %s on line %d