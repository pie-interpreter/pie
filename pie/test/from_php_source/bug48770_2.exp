%unicode|string%(27) "A::func2: This should work!"
%unicode|string%(27) "A::func3: This should work!"

Warning: call_user_func_array() expects parameter 1 to be a valid callback, cannot access private method A::func22() in %s on line %d

Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'A' does not have a method 'inexistent' in %s on line %d