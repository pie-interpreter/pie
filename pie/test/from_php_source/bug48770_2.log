
---- EXPECTED OUTPUT
%unicode|string%(27) "A::func2: This should work!"
%unicode|string%(27) "A::func3: This should work!"

Warning: call_user_func_array() expects parameter 1 to be a valid callback, cannot access private method A::func22() in %s on line %d

Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'A' does not have a method 'inexistent' in %s on line %d
---- ACTUAL OUTPUT
string(27) "B::func2: This should work!"
string(27) "B::func3: This should work!"

Warning: call_user_func_array() expects parameter 1 to be a valid callback, cannot access private method B::func22() in /home/sery0ga/work/php-src/Zend/tests/bug48770_2.php on line 22

Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'B' does not have a method 'inexistent' in /home/sery0ga/work/php-src/Zend/tests/bug48770_2.php on line 23
---- FAILED
