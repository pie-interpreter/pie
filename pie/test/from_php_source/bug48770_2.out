string(27) "B::func2: This should work!"
string(27) "B::func3: This should work!"

Warning: call_user_func_array() expects parameter 1 to be a valid callback, cannot access private method B::func22() in /home/sery0ga/work/php-src/Zend/tests/bug48770_2.php on line 22

Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'B' does not have a method 'inexistent' in /home/sery0ga/work/php-src/Zend/tests/bug48770_2.php on line 23